packages/playwright/src/util.ts

```diff
@@ -91,6 +91,12 @@ export function createFileFiltersFromArguments(args: string[]): TestFileFilter[]
   });
 }
 
+export function createProjectMatcher(projectNames?: string[], projectGrep?: string): Matcher {
+  const names = projectNames || [];
+  const regexp = projectGrep ? forceRegExp(projectGrep) : undefined;
+  return (project: string) => (!!names.length && names.includes(project)) || (!!regexp && regexp.test(project));
```

Comment:
The other place uses `toLowerCase` on the project names filter. I also wonder why do we have two separate filters in our codebase?

Reply:
Good question. Change this place to use filterProjects too.